BUILD.bazel

```diff
@@ -1,6 +1,7 @@
 load("@buildifier_prebuilt//:rules.bzl", "buildifier")
 load("@gazelle//:def.bzl", "gazelle")
 load("@rules_go//go:def.bzl", "TOOLS_NOGO", "go_binary", "go_library", "nogo")
+load("@rules_shell//shell:sh_binary.bzl", "sh_binary")
```

Comment:
Need to use rules for Shell due to the error `./BUILD.bazel:44: native-sh-binary: Function "sh_binary" is not global anymore and needs to be loaded from "@rules_shell//shell:sh_binary.bzl". (https://github.com/bazelbuild/buildtools/blob/main/WARNINGS.md#native-sh-binary)`